client_test.go

```diff
@@ -1310,6 +1308,50 @@ func Test_Client_InsertManyTx(t *testing.T) {
 	})
 }
 
+func Test_Client_JobGet(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+
+	type testBundle struct{}
+
+	setup := func(t *testing.T) (*Client[pgx.Tx], *testBundle) {
+		t.Helper()
+
+		config := newTestConfig(t, nil)
+		client := newTestClient(ctx, t, config)
+
+		return client, &testBundle{}
+	}
+
+	t.Run("FetchesAnExistingJob", func(t *testing.T) {
+		t.Parallel()
+
+		client, _ := setup(t)
+
+		newJob, err := client.Insert(ctx, noOpArgs{}, nil)
+		require.NoError(t, err)
+
+		job, err := client.JobGet(ctx, newJob.ID)
+		require.NoError(t, err)
+		require.NotNil(t, job)
+
+		require.Equal(t, newJob.ID, job.ID)
+		require.Equal(t, newJob.State, job.State)
+	})
+
+	t.Run("ReturnsErrNotFoundIfJobDoesNotExist", func(t *testing.T) {
+		t.Parallel()
+
+		client, _ := setup(t)
+
+		job, err := client.JobGet(ctx, 999999)
```

Comment:
Hah, very unlikely, but small chance this fails locally because our test DB sequences don't get reset between runs. Probably will never happen, but I guess zero might be a slighter safer alternative.

Reply:
Should only be an issue if a row actually exists with this ID, right? And we either truncate after each test or use a txn that gets rolled back.